elife/aws-cli.sls

```diff
@@ -0,0 +1,6 @@
+aws-cli:
+    cmd.run:
+        - name: pip install awscli
```

Comment:
does this make it available on the $PATH? I wonder how does it work in EC2 instances as it should already be installed, we will have to discover whether it installs another one or not. Maybe we can add:
```
        - unless:
            - aws
```